ffmuc-ipv6-ra-filter/files/usr/lib/micron.d/ffmuc-ipv6-ra-filter

```diff
@@ -0,0 +1 @@
+* * * * * sleep $(awk 'BEGIN{srand();print int(rand()*40)}') && /lib/gluon/ffmuc-ipv6-ra-filter/ipv6-ra-filter
```

Comment:
This is not a good solution. At this point, It's much more practical to have a procd service running than executing a cronjob every minute.

Reply:
I would be very interested in how this can be done. Currently, having a cronjob every minute fits the codestyle of the community-packages repo though ;)
But this is due to not knowing the proper solution.

Is there an example of a procd job run every minute in another package somewhere?